packages/astro/src/core/render/result.ts

```diff
@@ -131,157 +80,3 @@ class Slots {
 		return outHTML;
 	}
 }
-
-export function createResult(args: CreateResultArgs): SSRResult {
```

Comment:
Why was this function moved out?

Reply:
So that the code for the Astro global could live alongside that of APIContext and share the implementation details.